### DISCUSSION TITLE
How to distinguish access rights for different services?

### DISCUSSION THREAD

**Discussion by Northguy** [Tecnativa/docker-socket-proxy#48] [Q&A]
Suppose I am running Multiple services that each require their own unique set of access rights to the docker API. Is there a way to configure this in the socket-proxy, or do I have to configure multiple socket proxy instances, each with their unique access rights and access the API through these separate instances?

**Comment by joao-p-marques** [comment]
Yes, you'd need a different instance of docker-socket-proxy for each set of permissions. 
This is because of the way we limit access with HAProxy through environment variables. It is easier to configure, though 🙂
